pkgs/tools/wayland/hyprevents/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchgit, lib, makeWrapper, stdenv, hyprland, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "hyprevents";
+  version = "1.0-20221209";
+
+  src = fetchgit {
+    url = "https://github.com/vilari-mickopf/hyprevents";
+    rev = "95426ae4dac70c6e6ed67918dbf3bdedfdf28f37";
+    sha256 = "sha256-SLt2ukCJbfkvnGLgy9x7/3BoI2OqC4VIdgM5z768EXc=";
+  };
+
+  patches = [ ./0001-move-event_loader-event_handler-make-it-executable.patch ];
+
+  strictDeps = true;
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ bash ];
+
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+
+    export PREFIX="$out"
+    make install
+
+    wrapProgram "$out/bin/hyprevents" \
+      --prefix PATH : "$out/bin:${lib.makeBinPath [ hyprland ]}"
+
+    runHook postInstall
+  '';
+  passthru.scriptName = "hyprevents";
+
+  meta = with lib; {
+    description = "Invoke shell functions in response to Hyprland socket2 events. Forked from hyprwm.";
+    homepage = "https://github.com/vilari-mickopf/hyprevents";
+    maintainers = with maintainers; [ ];
```

Comment:
Are you willing to maintain this package? If so, you can add yourself as a maintainer